src/apps/material/material.test.ts

```diff
@@ -174,6 +174,24 @@ describe("Material", () => {
       .click();
   });
 
+  it("Renders reviews", () => {
+    cy.interceptGraphql({
+      operationName: "getMaterial",
+      fixtureFilePath: "material/fbi-api.json"
+    });
+    cy.interceptGraphql({
+      operationName: "getReviewManifestations",
+      fixtureFilePath: "material/reviews.json"
+    });
+    cy.visit("/iframe.html?id=apps-material--default&viewMode=story&type=bog");
+
+    cy.getBySel("material-reviews-disclosure").should("be.visible").click();
+    cy.getBySel("material-reviews").should(
+      "contain",
```

Comment:
NB: We probably need some scrolling here when the performance PR has been merged, because of lazy loading..

Reply:
Thanks for heads up! :)